mkt/lookup/tests/test_views.py

```diff
@@ -1211,3 +1212,22 @@ def test_by_name(self):
     def test_by_id(self):
         data = self.search(q=self.website.pk)
         self.verify_result(data)
+
+
+class TestWebsiteEdit(mkt.site.tests.TestCase):
```

Comment:
The view is actually pretty empty, most of the logic is in the form, which is also where the tests are. But if you have something specific I'd be happy to add it. Perhaps just a simple test that a form submission redirects and updates the data as a view test?